config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_1M.yml

```diff
@@ -14,9 +14,5 @@ implicit_diffusion: true
 approximate_linear_solve_iters: 2
 rad: "clearsky"
 dt_rad: "6hours"
-job_id: "longrun_1M"
+job_id: "longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_1M"
 toml: [toml/longrun_1M.toml]
```

Comment:
Do you want to change the toml file name too?

Reply:
Sure!